### ISSUE TITLE
Not working with ngx-translate

### ISSUE THREAD

**Issue by iCodr8** [tnicola/ngx-joyride#73]
We are using the ngx-translate module to translate all contents of the angular app. The problem is, that it is not working with the `title` and `text` attributes of ngx-joyride.

**To Reproduce**
```
<div joyrideStep="dashboardStatistics"
         [title]="'TOUR.DASHBOARD_STATISTICS.TITLE' | translate"
         [text]="'TOUR.DASHBOARD_STATISTICS.TEXT' | translate">
    ... more content ...
</div>
```

**Expected behavior**
The translated text should be visible in the tour window.

**Screenshots**
<img width="322" alt="Bildschirmfoto 2019-09-20 um 15 05 01" src="https://user-images.githubusercontent.com/1264033/65329161-31f33a80-dbb8-11e9-83d4-f75e09a4f314.png">

**Details:**
 - Browser chrome 76
 - @angular/core 7.2.15
 - ngx-joyride 2.2.10
 - ngx-translate 11.0.1

**Comment by rtcimi** [comment]
I was/am using it like this:

<div joyrideStep="dashboardStatistics"
         [title]="TOUR.DASHBOARD_STATISTICS.TITLE"
         [text]="TOUR.DASHBOARD_STATISTICS.TEXT">
    ... more content ...
</div>

**Comment by iCodr8** [comment]
No, that's not working. I want the translated string and not the translation key.

**Comment by shifenis** [comment]
I'm using `ngx-translate` too and I can't reproduce your error

**Comment by iCodr8** [comment]
I think it is a timing problem. Sometimes it will be shown, but the most time not.

**Comment by iCodr8** [comment]
I have found the problem! Joyride is not working with observables or async operations.
The translation is loaded after some milliseconds and joyride uses the empty value before. When I am adding an if condition than it is working, because joyrideStep will be initialized only after the if-condition is true (translation loaded).

But this is a bad solution. Can this be fixed by joyride please?

```
<div *ngIf="'TOUR.DASHBOARD_STATISTICS.TITLE' | translate"
         joyrideStep="dashboardStatistics"
         [title]="'TOUR.DASHBOARD_STATISTICS.TITLE' | translate"
         [text]="'TOUR.DASHBOARD_STATISTICS.TEXT' | translate">
    ... more content ...
</div>
```

**Comment by tnicola** [comment]
Hi @iCodr8 , 

I've tried but I'm not able to reproduce your bug. Could you please reproduce it on a small app?
